qa/restart-upgrade/src/test/resources/processor/IndexMappingSingleShard.json

```diff
@@ -0,0 +1,34 @@
+{
+  "settings": {
+    "index": {
+      "knn": true,
+      "refresh_interval": "30s",
+      "default_pipeline": "%s"
```

Comment:
is this default pipeline for data ingest? where do we use it in BWC tests?

Reply:
yes, I have simulated the same from opensearch official documentation

Reply:
but where do we use it in tests? If we need it for something it's ok. Typically we're using ingest processor for getting embeddings and then store them

Reply:
We use this for ingestion and generate embeddings